### DISCUSSION TITLE
Unable to CREATE MODEL on Windows using Anaconda Prompt

### DISCUSSION THREAD

**Discussion by roberanegussie** [mindsdb/mindsdb#5653] [Get help] [answered]
Hello, I am new to mindsdb. I installed mindsdb locally using anaconda prompt on Windows 10 following the stated guidelines
in the documentation. I was then able to connect it to the mysql server I have setup locally. I was also able to query a simple 
select statement on the database. However, when trying to **_create a model_**  as stated in the documentation I encountered an error.
``` ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'model mindsdb.mvp_mod [from playoff(select MPG, Ftperc, 2Pperc, 3Pperc, eFGpe' at line 1```  

The statement I had run was as follows:
```create model mindsdb.mvp_mod [from playoff(select MPG, Ftperc, 2Pperc, 3Pperc, eFGperc, PPG, RPG, APG, SPG from stats202223)] predict rank; ```

**Comment by ZoranPandovski** [comment, marked as answer]
Hi @roberanegussie, it looks like you have used the wrong syntax. Please check the MindsDB docs for:

1. Create database integration https://docs.mindsdb.com/sql/create/databases
2. Create a model https://docs.mindsdb.com/sql/create/predictor
